dags/veda_data_pipeline/veda_generic_vector_pipeline.py

```diff
@@ -0,0 +1,92 @@
+import pendulum
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.models.variable import Variable
+from airflow.providers.amazon.aws.operators.ecs import EcsRunTaskOperator
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.utils.trigger_rule import TriggerRule
+from airflow.models.variable import Variable
+
+from veda_data_pipeline.groups.processing_tasks import build_generic_vector_kwargs
+from veda_data_pipeline.groups.discover_group import discover_from_s3_task, get_files_to_process
+
+
+dag_doc_md = """
+### Generic Ingest Vector
+#### Purpose
+This DAG is supposed to be triggered by `veda_discover`. But you still can trigger this DAG manually or through an API 
```

Comment:
```suggestion
This DAG is used to ingest vector data for use in the VEDA Features API
```

I don't think the old docstring still applies.